src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.AspNet.Mvc.TagHelpers
     public class FormTagHelper : TagHelper
     {
         private const string ActionAttributeName = "asp-action";
-        private const string AntiForgeryAttributeName = "asp-anti-forgery";
+        private const string AntiforgeryAttributeName = "asp-anti-forgery";
```

Comment:
Since you're such an "anti" hater, why didn't you rename this attribute `asp-antiforgery`. Looks inconsistent as-is.